doc/rules/phpdoc/general_phpdoc_annotation_remove.rst

```diff
@@ -52,3 +52,21 @@ With configuration: ``['annotations' => ['package', 'subpackage']]``.
      * @version 1.0
      */
     function foo() {}
+
+Example #3
+~~~~~~~~~~
+
+With configuration: ``['annotations' => ['template()']]``,he won't do anything because it doesn't support symbols. Please follow this regular expression ``@[a-zA-Z0-9_-]+(?=\s|$)``.
```

Comment:
maybe we should rather add the possibility to drop the `template()` and bring docs example how to configure it?